src/index/generateTrees.ts

```diff
@@ -6,8 +6,10 @@ import { RootOption as Root } from "./shared/RootOption";
 import { buildGraph } from "./generateTrees/buildGraph";
 import { findEntryPoints } from "./generateTrees/findEntryPoints";
 import { toFractalTree } from "./generateTrees/toFractalTree";
+import { detectLonelyFiles } from "./shared/detect-lonely-files";
+import { Config } from "../index";
 
-export function generateTrees(restructureMap: { [key: string]: string[] }) {
+export function generateTrees(restructureMap: { [key: string]: string[] }, {renameOnlyChild}: Config) {
```

Comment:
https://github.com/benawad/destiny/pull/89#discussion_r386085311
```suggestion
export function generateTrees(restructureMap: { [key: string]: string[] }, { avoidSingleFile }: Config) {
```